packages/utils/test/schema/getDefaultFormStateTest.ts

```diff
@@ -34,7 +34,7 @@ export default function getDefaultFormStateTest(testValidator: TestValidatorType
           },
           $ref: '#/definitions/testdef',
         };
-        expect(computeDefaults(testValidator, schema, undefined, schema)).toEqual({
+        expect(computeDefaults(testValidator, schema, { rootSchema: schema })).toEqual({
```

Comment:
I wasn't sold the new function signature for `computeDefaults` until I saw these tests. This is much cleaner!

Reply:
With all the new optional stuff being added between my MR and the others going in ahead of me I figured it just made sense. Glad you like it!